package com.hackbulgaria.exceptions1;

public class MainFunc {
    public static void main(String[] args) {
        try{
            boo(0);
        }catch(Exception e){
            System.out.println(e.toString());
            System.out.println(e.getMessage());
            e.printStackTrace();
        }
    }
    
    public static void boo(int a){
        foo();
    }
    public static void foo(){
        if(1 < 2){
            throw new DatabaseCorruptedException();
        }
    }

}
